apps/crn-frontend/src/network/groups/GroupProfile.tsx

```diff
@@ -68,7 +68,9 @@ const GroupProfile: FC<GroupProfileProps> = ({ currentTime }) => {
           About={() => (
             <About group={group} groupTeamsElementId={groupTeamsElementId} />
           )}
-          Calendar={() => <Calendar calendars={group.calendars} />}
+          Calendar={() => (
```

Comment:
Why is this a function?

Reply:
I've made this way so ProfileSwitch can receive `Calendar` directly without having to receive the props Calendar receive :)